src/parser/druid/restoration/SpellInfo.ts

```diff
@@ -23,7 +24,7 @@ const DEFAULT_INFO = { // we assume unlisted spells scale with vers only (this w
   vers: true,
 };
 
-export const DRUID_HEAL_INFO = {
+export const DRUID_HEAL_INFO: ListOfHealerSpellInfo = {
```

Comment:
TBH i don't know if this is needed or not. Since other healers are not breaking I don't think it is but thoughts?